### PR TITLE
Db mk 2246 publish duplicate

### PR DESCRIPTION
**Pull request by kimminahm**
- [ ] Dependent on backend https://github.com/fedspendingtransparency/data-act-broker-backend/pull/947
- [x] Reviewed by frontend

### CODE REVIEW

File: src/js/components/uploadDetachedFiles/UploadDetachedFileValidation.jsx [hunk 1]
```diff
@@ -341,6 +343,10 @@ class UploadDetachedFileValidation extends React.Component {
 				validationButton = <button className='pull-right col-xs-3 us-da-disabled-button' disabled>You do not have permissions to publish</button>;
 			}
 		}
+
+		if (this.state.error !== 0) {
```
Comment (alburde1) [inline src/js/components/uploadDetachedFiles/UploadDetachedFileValidation.jsx]: Why was this moved down here instead of being left in the else if block?

File: src/js/components/uploadDetachedFiles/UploadDetachedFileValidation.jsx [hunk 2]
```diff
@@ -102,14 +103,11 @@ class UploadDetachedFileValidation extends React.Component {
...
-				var submission = true;
-				if(response.publish_status==='published'){
-					submission = false;
-				}
+
 				const job = Object.assign({}, this.state.jobResults);
 				job.detached_award = response.jobs[0];
 
-				if (this.state.published && response.fabs_meta.publish_date && this.dataTimer) {
+				if (this.state.published !== 'publishing' && response.fabs_meta.publish_date && this.dataTimer) {
```
[the remark this answers is not in the document]
Comment (kimminahm) [reply src/js/components/uploadDetachedFiles/UploadDetachedFileValidation.jsx]: if it is unpublished it shouldnt keep checking, it should kill the loop so this works as expected

File: src/js/components/uploadDetachedFiles/UploadDetachedFileValidation.jsx (hunk 1 above)
Comment (kimminahm) [reply src/js/components/uploadDetachedFiles/UploadDetachedFileValidation.jsx]: the logic above it is for if it is not publishing. It would change states before so this was the simplest solution I saw
